### DISCUSSION TITLE
Suggestions: contract info and class-specific desired item mods

### DISCUSSION THREAD

**Discussion by worldendDominator** [Lailloken/Exile-UI#174] [Ideas & Feedback]
I have two ideas.

First, extend map info to contracts too. While I'm not sure how useful it would be to most players (it's definitely useful to me though), but it sounds like something easy to implement.

Second, I'd prefer to have desired mods in item info to be separate for different categories - for example, resistances on armor and jewelry are good, on weapons often bad (wasting space for much more important mods), on jewels acceptable but not something I'm looking out for. Taking it further, local damage is useless on caster weapons but essential on regular weapons; although at this point I'm not sure if it's too much fine detail to be generally usable. It could be a toggle in options though.

What do you think about these?

**Comment by Lailloken** [comment]
1. The map-info panel is something I want to re-do in the near future, and I can also look into implementing contract support. Sounds good.
2. More specific highlighting was something I wanted to include from the beginning but I then ran into the same question of how detailed it could/should be, so I delayed that.

My current plans are to implement an item-class-specific highlighting system that takes precedence over the global one. Example: If someone has flat ele damage highlighted as desirable right now, those mods are highlighted green on all items. Long-rightclicking flat ele damage on a wand would mark it undesired for wands only. Inversely, if someone only wanted to highlight attack speed on gloves, they can long-click it to only be highlighted on those but nothing else. The "issue" is that this override has to be repeated for each caster item-class, so the alternative would be to group these together into a single large caster group (but this would make the system as a whole less flexible). I guess I can make grouping item-classes an option.

Feel free to add anything if you have more ideas.

Thanks for your suggestions.
